js/app.js

```diff
@@ -78,49 +76,58 @@ angular.module('PathOfDamage', [])
     if (hit === null) {
       return {};
     }
-    var physDamage = hit;
+
+    var damage = {
```

Comment:
Yeah, it just changes so often. I think when I do the revamping of dataServices with prototypes each table will know how to add its own total to the damage object, so this method will become much shorter.